app_dart/lib/src/request_handlers/refresh_cirrus_status.dart

```diff
@@ -121,7 +121,12 @@ Future<List<dynamic>> queryCirrusGraphQL(
   if (result.data == null) {
     return tasks;
```

Comment:
Maybe also log if data is missing. 🤔

Reply:
Hey @fkorotkov!
Can do! 
After merging this and checking the logs it looks like there wasn't an issue with the query.
The change that caused the issue was https://github.com/flutter/flutter/pull/51919, which didn't have any cirrus tasks scheduled to run, which is actually working as intended. No cirrus tasks were run for that change since we run tests conditionally, and it didn't change any relevant files to do so. Do you think the query `result.data` should have been null if there wasn't a build?

Reply:
`data` can be null if there is `error` so if everything works as expected you should be good.

Glad queries worked as expected!